### DISCUSSION TITLE
file-roller and RAR files

### DISCUSSION THREAD

**Discussion by picaron77** [SpiralLinux/SpiralLinux-project#473] [General]
file roller doesn't work well in spiral cinnamon, it fails to unzip rar files. I have all the decompression files installed. I had to use Engrampa , but I can't set it as the default preferred one, so I can use it with the context menu.

**Comment by geckolinux** [comment]
Hi there, I know this sort of "works for me" reply is frustrating but... it works for me. ;-) I use SpiralLinux Cinnamon on my personal systems and I just tested file-roller with several RAR files and they all worked.
